main.go

```diff
@@ -171,7 +171,24 @@ func main() {
 		setupLog.Error(err, "unable to set up health check")
 		os.Exit(1)
 	}
-	if err := mgr.AddReadyzCheck("readyz", healthz.Ping); err != nil {
+
+	// console plugin is dependent on CRDs that are part of these CSVs and we
+	// can't check for only odf-dependencies CSV as OLM doesn't guarantee
+	// (based on observations) existence of all CRDs brought by OLM dependency
+	// mechanism.
```

Comment:
``` suggestion
// The console plugin depends on CRDs provided by these CSVs. 
// Checking only the odf-dependencies CSV is insufficient, 
// because, based on observations, the presence of all required CRDs 
// is not guaranteed even when the odf-dependencies CSV is in a succeeded state.
```